doc/user-guide.md

```diff
@@ -333,6 +335,39 @@ spec:
 
 ### Service binding
 
+#### Service Binding Operator
+
+The [Service Binding Operator](https://github.com/redhat-developer/service-binding-operator) enables application developers to bind applications together with operator-managed backing services. This can be achieved by creating a `ServiceBindingRequest` custom resource.
+
+In order for the Runtime Component Operator to pass the binding information to the application, you can specify the name of the `ServiceBindingRequest` as follows:
+
+```yaml
+apiVersion: app.stacks/v1beta1
+kind: RuntimeComponent
+metadata:
+  name: my-app
+spec:
+  applicationImage: quay.io/my-repo/my-app:1.0
+  bindings:
+    resourceRef: my-binding
+```
+
+Once the Service Binding Operator processes the `my-binding` custom resource of the type `ServiceBindingRequest`, it creates a `Secret` containing binding information provided by the backing service. The Runtime Component Operator would inject the `Secret` as environment variables into application pods.
```

Comment:
Change "Once the Service Binding" to "After the Service Binding".
Put the correct noun after `ServiceBindingRequest` (such as "custom resource", etc)
Put the correct noun after  "`Secret`". I believe that "`Secret`" is an object. If it is, the wording would be "a `Secret` object".
Here is my attempt at rewording the paragraph based on my comments. Feel free to adjust as I might have picked the wrong noun for various terms. Slack me to discuss if need be.
"After the Service Binding Operator processes the `my-binding` custom resource of the type `ServiceBindingRequest`, it creates a `Secret` object. The `Secret` object contains binding information that the backing service provides. The Runtime Component Operator injects the `Secret` object as environment variables into application pods.".